Not suitable for upstream

Hardcode the NSS/NSPR includes and libs

--- a/CMakeLists.txt	2017-11-09 12:18:43.977702740 -0800
+++ b/CMakeLists.txt	2017-11-09 12:28:32.390516702 -0800
@@ -928,7 +928,11 @@
 # subparts flags
 # ******************************
 
-pkg_check_modules(DATA_SERVER REQUIRED gio-2.0 gmodule-2.0 libsecret-1>=${libsecret_minimum_version} libxml-2.0 libsoup-3.0 ${mozilla_nspr} ${mozilla_nss})
+pkg_check_modules(DATA_SERVER REQUIRED gio-2.0 gmodule-2.0 libsecret-1>=${libsecret_minimum_version} libxml-2.0 libsoup-3.0)
+set(WITH_NSPR_INCLUDES "/usr/include/mps")
+set(WITH_NSS_INCLUDES "/usr/include/mps")
+set(WITH_NSPR_LIBS "/usr/lib/mps/64")
+set(WITH_NSS_LIBS "/usr/lib/mps/64")
 set(DATA_SERVER_INCLUDE_DIRS ${DATA_SERVER_INCLUDE_DIRS} ${MANUAL_NSPR_INCLUDES} ${MANUAL_NSS_INCLUDES})
 set(DATA_SERVER_LDFLAGS ${DATA_SERVER_LDFLAGS} ${MANUAL_NSPR_LIBS} ${MANUAL_NSS_LIBS})
 
@@ -941,7 +945,7 @@
 set(CALENDAR_CFLAGS ${CALENDAR_CFLAGS} -DLIBICAL_GLIB_UNSTABLE_API=1)
 
 pkg_check_modules(CAMEL REQUIRED gio-2.0 gmodule-2.0 ${mozilla_nss} ${mozilla_nspr} sqlite3>=${sqlite_minimum_version})
-set(CAMEL_CFLAGS ${CAMEL_CFLAGS} ${KRB5_CFLAGS} ${MANUAL_NSS_INCLUDES} ${MANUAL_NSPR_INCLUDES} ${ICU_CFLAGS})
+set(CAMEL_CFLAGS ${CAMEL_CFLAGS} ${KRB5_CFLAGS} ${MANUAL_NSS_INCLUDES} ${MANUAL_NSPR_INCLUDES} ${ICU_CFLAGS} -L/usr/lib/mps/64 -lnss3 -lnspr4 -R/usr/lib/mps/64) 
 set(CAMEL_LDFLAGS ${CAMEL_LDFLAGS} -lz ${KRB5_LDFLAGS} ${MANUAL_NSS_LIBS} ${MANUAL_NSPR_LIBS} ${ICU_LDFLAGS})
 
 if(NOT (MANUAL_NSPR_INCLUDES STREQUAL ""))
